scipy/interpolate/interpnd.pyx

```diff
@@ -875,7 +875,7 @@ class CloughTocher2DInterpolator(NDInterpolatorBase):
     continuously differentiable.
 
     The gradients of the interpolant are chosen so that the curvature
-    of the interpolating surface is approximatively minimized. The
+    of the interpolating surface is approximately minimized. The
```

Comment:
This shouldn't have been changed, and we've already had that [discussion](https://github.com/scipy/scipy/pull/19041#discussion_r1335258728). The statement is not the same; one describes a process, the other a result. Please revert this change, and stop re-introducing it. Math sometimes uses language that's unusual in other contexts, and it's codespell that has to adapt to this, not math.

Reply:
Sorry about that.

Reply:
Would you agree to adding a codespell config in pyproject.toml and a file with codespell false positives?

Reply:
Thanks for fixing this quickly.

> Would you agree to adding a codespell config in pyproject.toml and a file with codespell false positives?

That depends on the other maintainers; I won't object, but I'm not enthusiastic about adding this to our pyproject.toml, which so far is quite strictly limited to project metadata and what's necessary for building wheels.

On top of that, my argument remains that accepting correct spellings is the responsibility codespell, and it shouldn't fall to us to carry "false positives" for stuff that codespell flags incorrectly. While I haven't invested time in finding out how codespell works under the hood, my understanding is that this would be fixed by telling codespell not to warn on words in the "rare" dictionary.

That sounds like the best solution to me _by far_, and so I'd ask you to either try that first, or provide an argument why it's not possible.

Reply:
Nowadays it is certainly best to use spellchecking based on deep learning and a large corpus of N-grams. By the way, this would raise the question of how to split "words" in a more acute way than in most natural languages. Any way, codespell checks individual words, so we need to find a balance between too many false positives and too many false negatives. I feel it is hard to get absolutely no false positives, some manual adjustments might be necessary.
If you would rather keep `pyproject.toml` clean, the solution would be a `.codespellrc` file. As for the contents of the config, they could leave out the dictionary of typos that are rare words, or manually create a list of false positives to ignore.

Reply:
You can write a proposal to [the mailing list](https://mail.python.org/archives/list/scipy-dev@python.org/) if you would like to. Do you envision this being a check in CI, or more (somewhat) regular PRs like this one? Either way, I imagine that at least some maintainers will object to it being a good use of time, but feel free to give your best argument if you are confident that it is worth it.